pkg/yurthub/filter/interfaces.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 type Interface interface {
+	Name() string
```

Comment:
Is the change in the interface necessary?

Reply:
@Fei-Guo ok, i have removed the new func `Name()`.